Library/Formula/mitmproxy.rb

```diff
@@ -19,88 +21,84 @@ class Mitmproxy < Formula
   depends_on "protobuf" => :optional
 
   resource "pyopenssl" do
-    url "https://pypi.python.org/packages/source/p/pyOpenSSL/pyOpenSSL-0.14.tar.gz"
-    sha1 "eb51f23f29703b647b0f194beaa9b2412c05e0f6"
+    url "https://pypi.python.org/packages/source/p/pyOpenSSL/pyOpenSSL-0.15.1.tar.gz"
+    sha256 "f0a26070d6db0881de8bcc7846934b7c3c930d8f9c79d45883ee48984bc0d672"
   end
-
```

Comment:
Can we keep the spaces between the resources? Formulae are a nightmare to read when they're all bunched up.